src/javascript/_common/language.js

```diff
@@ -9,7 +9,7 @@ const Language = (() => {
     const all_languages = {
         ACH  : 'Translations',
         EN   : 'English',
-        DE   : 'Deutsch',
+        // DE   : 'Deutsch',
```

Comment:
can you add a todo comment to uncomment both of these when we want to enable german again?